demo.md

```diff
@@ -0,0 +1,23 @@
+## Demo script:
+
+- ERC-4337 compatible Smart Contract wallet
+- Unique feature: gasless off-chain txs with instant finality enabled by HTLC technology, as used in Bitcoin Lightning. Funds can be moved in and out of the off-chain payment system without any on-chain txs!
```

Comment:
I think we should lean into the cross-chain aspect and also emphasize that. Not only do the payments happen completely off-chain, but the payments can be redeemed on different chains.